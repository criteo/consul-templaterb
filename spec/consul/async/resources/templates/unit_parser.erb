KV Parse results
JSON
It should display: {"toto":{"tata":"titi"}}
<%=JSON.generate(kv('json/test.json').get_value_json)%>

YAML
It should display:
---
this:
  is:
    only:
      a:
        test: true

<%=YAML.dump(kv('yaml/test.yml').get_value_yaml)%>