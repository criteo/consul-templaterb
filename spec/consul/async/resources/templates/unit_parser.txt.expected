KV Parse results
JSON
It should display: {"toto":{"tata":"titi"}}
{"toto":{"tata":"titi"}}

YAML
It should display:
---
this:
  is:
    only:
      a:
        test: true

---
this:
  is:
    only:
      a:
        test: true
